optking/opt_helper.py

```diff
@@ -567,13 +571,13 @@ def __init__(self, optimization_input, **kwargs):
         self.computer = optwrapper.make_computer(self.opt_input, "qc")
         self.computer_type = "qc"
         self.build_coordinates()
-        print(type(self.params))
         self.opt_manager = OptimizationManager(self.molsys, self.history, self.params, self.computer)
 
     @classmethod
     def from_dict(cls, d):
         helper = super().from_dict(d)
         helper.computer = compute_wrappers.make_computer_from_dict("qc", d.get("computer"))
+        helper.opt_manager = OptimizationManager.from_dict(d["opt_manager"], helper.molsys, helper.history, helper.params, helper.computer)
```

Comment:
Please check the comment block at the top of opt_helper.py and either update it, or remove it to the extent that it's inaccurate.